ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml

```diff
@@ -223,12 +222,15 @@
 
     <date-comparison inline-template>
         <div>
-            @* TODO: Implement this as non-javascript friendly *@
-            <ul class="text-warning" id="date-warnings">
-                <li v-for="warning in dateWarnings" v-if="warning">
-                    <h4>{{ warning.message }}</h4>
-                </li>
-            </ul>
+            <div v-if="dateWarnings.length > 0">
+                <govuk-warning-text id="nhs-number-warning" ref="nhs-number-warning">
+                    <div class="warning-container">
+                        <p v-for="warning in dateWarnings" v-if="warning">
+                            <strong>{{ warning.message }}</strong>
+                        </p>
+                    </div>
+                </govuk-warning-text>
+            </div>
```

Comment:
Spoke to Jan briefly about this: as it could be considered a progressive enhancement - happy for this to remain not non-javascript friendly. However I will raise a tech debt ticket to add the warnings on mount.